views/groupDetails.njk

```diff
@@ -163,7 +163,8 @@
     }) }}
     {% endif %}
 
-    {% set searchUrl = '/search-external-users/results?groupCode='+ groupDetails.groupCode %}
+    {% set searchUrl = '/search-external-users?user=&status=ALL&roleCode=&groupCode='+ groupDetails.groupCode %}
+
```

Comment:
I'd be tempted to just pass through the groupCode.  It should just pick up the others as default